pipeline/init.bats

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bats
+
+set -ueo pipefail
+
+setup() {
+    load "bats-helpers/bats-support/load"
+    load "bats-helpers/bats-assert/load"
+
+    export PATH="${BATS_TEST_DIRNAME}/../bin:${PATH}"
+
+    export CK8S_CONFIG_PATH="${BATS_TMPDIR}/ck8s-apps-config"
+    export CK8S_ENVIRONMENT_NAME=test
+    export CK8S_PGP_FP=529D964DE0BBD900C4A395DA09986C297F8B7757
+    export CK8S_CLOUD_PROVIDER=aws
+}
+
+teardown() {
+    rm -rf "${CK8S_CONFIG_PATH}"
```

Comment:
Maybe I'm a bit paranoid here, but this feels very dangerous if you've set the variable wrong and run this function.
I'd rather have `${BATS_TMPDIR}/ck8s-apps-config` here as well even though it adds duplicates.

Reply:
I agree

Reply:
Good point. :smile:

In any case, it's advisable to run the whole Bats suite inside a container because it does not give guarantees about side-effects.

Reply:
Fixed, PTAL.